src/direct/direct-export-buf.c

```diff
@@ -54,9 +54,44 @@ bool direct_initExporter(NVDriver *drv) {
 
     //make sure we have a drm fd
     if (drv->drmFd == -1) {
-        //TODO make this configurable
-        drv->drmFd = open("/dev/dri/renderD128", O_RDWR|O_CLOEXEC);
-        LOG("Manually opened DRM device");
+        uint8_t nvdGpu = 0;
+        char *nvdGpuStr = getenv("NVD_GPU");
```

Comment:
This value is already be available as `drv->cudaGpuId`. That variable might be set to `-1`, meaning the default GPU, in this case we can change it to `0`.